sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/TableOutputResolver.scala

```diff
@@ -288,11 +287,13 @@ object TableOutputResolver extends SQLConfHelper with Logging {
   private def resolveColumnsByPosition(
       tableName: String,
       inputCols: Seq[NamedExpression],
-      expectedCols: Seq[Attribute],
```

Comment:
the previous name looks more correct

Reply:
Thought about reduce LOC before, addressed